docs/architecture/data-model/erd-governance-consent.html

```
@@ -46,81 +46,128 @@ <h1>Audit events, consent receipts, takedowns</h1>
+      <text x="1024" y="642" fill="#525252" font-size="10" font-family="'Space Mono', monospace" text-anchor="end">varchar</text>
+      <text x="576" y="660" fill="#1A1A1A" font-size="10" font-family="'Space Mono', monospace">policy_version</text>
+      <text x="1024" y="660" fill="#525252" font-size="10" font-family="'Space Mono', monospace" text-anchor="end">varchar</text>
+      <text x="576" y="678" fill="#1A1A1A" font-size="10" font-family="'Space Mono', monospace">source</text>
+      <text x="1024" y="678" fill="#525252" font-size="10" font-family="'Space Mono', monospace" text-anchor="end">varchar</text>
+      <text x="576" y="696" fill="#1A1A1A" font-size="10" font-family="'Space Mono', monospace">locale</text>
+      <text x="1024" y="696" fill="#525252" font-size="10" font-family="'Space Mono', monospace" text-anchor="end">varchar</text>
+      <text x="576" y="714" fill="#1A1A1A" font-size="10" font-family="'Space Mono', monospace">status</text>
+      <text x="1024" y="714" fill="#525252" font-size="10" font-family="'Space Mono', monospace" text-anchor="end">varchar</text>
+      <text x="576" y="732" fill="#1A1A1A" font-size="10" font-family="'Space Mono', monospace">given_at</text>
+      <text x="1024" y="732" fill="#525252" font-size="10" font-family="'Space Mono', monospace" text-anchor="end">timestamptz</text>
+      <text x="576" y="750" fill="#1A1A1A" font-size="10" font-size="10" font-family="'Space Mono', monospace">withdrawn_at</text>
```

HTMLHint (1.9.2)

[error] Duplicate of attribute name [ font-size ] was found.

(attr-no-duplication)